test/parallel/test-cluster-server-restart-rr.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const cluster = require('cluster');
+
+cluster.schedulingPolicy = cluster.SCHED_RR;
+
+if (cluster.isMaster) {
+  const worker1 = cluster.fork();
+  worker1.on('listening', common.mustCall(() => {
+    const worker2 = cluster.fork();
+    worker2.on('exit', (code) => {
```

Comment:
Could this be a `common.mustCall()`? It might be good to test `worker1`'s exit code too.